python/ray/tests/test_webui.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import sys
+import subprocess
+import pytest
+import ray
+
+
+@pytest.mark.skipif(sys.version_info < (3, 5, 3),
+    reason="requires python3.5.3 or higher")
+def test_get_webui():
+    # Try to install dependencies for this unittest.
+    # "aiohttp" requires that Python version >= 3.5.3.
+
+    try:
+        import aiohttp
+        import psutil
+    except (ImportError, ModuleNotFoundError):
+        # We install them here because they require python3,
+        # and they are not used by other unittests.
+        subprocess.check_call(
+            [sys.executable, "-m", "pip", "install", "aiohttp", "psutil"])
```

Comment:
I don't think it's a good idea to install packages at runtime.
It's better to install them [here](https://github.com/ray-project/ray/blob/master/ci/travis/install-dependencies.sh#L51), like:
```shell
if is_python3:
pip install aiohttp==x.x.x psutil====x.x.x
fi
```